editor.planx.uk/src/@planx/components/Pay/Editor.tsx

```diff
@@ -1,225 +1,392 @@
+    >
+      {({ values, handleChange, setFieldValue, errors }) => (
+        <Form id="modal" name="modal">
+          <ModalSection>
+            <ModalSectionContent title="Payment" Icon={ICONS[TYPES.Pay]}>
+              <InputRow>
+                <Input
+                  format="large"
+                  placeholder="Page title"
+                  name="title"
+                  value={values.title}
+                  onChange={handleChange}
+                />
+              </InputRow>
+              <InputRow>
+                <Input
+                  format="bold"
+                  placeholder="Banner title"
+                  name="bannerTitle"
+                  value={values.bannerTitle}
+                  onChange={handleChange}
+                />
+              </InputRow>
+              <InputRow>
+                <RichTextInput
+                  placeholder="Banner description"
+                  name="description"
+                  value={values.description}
+                  onChange={handleChange}
+                />
+              </InputRow>
+              <InputRow>
+                <Input
+                  format="data"
+                  name="fn"
+                  value={values.fn}
+                  placeholder="Data field for calculated amount"
+                  onChange={handleChange}
+                />
+              </InputRow>
+            </ModalSectionContent>
+            <ModalSectionContent>
+              <InputRow>
+                <Input
+                  format="large"
+                  placeholder="Instructions title"
+                  name="instructionsTitle"
+                  value={values.instructionsTitle}
+                  onChange={handleChange}
+                />
+              </InputRow>
+              <InputRow>
+                <RichTextInput
+                  placeholder="Instructions description"
+                  name="instructionsDescription"
+                  value={values.instructionsDescription}
+                  onChange={handleChange}
+                />
+              </InputRow>
+            </ModalSectionContent>
+            <OptionButton
+              selected={values.hidePay}
+              onClick={() => {
+                setFieldValue("hidePay", !values.hidePay);
+              }}
+              style={{ width: "100%" }}
+            >
+              Hide the pay buttons and show fee for information only
+            </OptionButton>
+          </ModalSection>
+          <ModalSection>
+            <ModalSectionContent
+              title="GOV.UK Pay Metadata"
+              Icon={DataObjectIcon}
+            >
+              <Typography variant="subtitle2" sx={{ mb: 2 }}>
+                Include metadata alongside payments, such as VAT codes, cost
+                centers, or ledger codes. See{" "}
+                <Link href="#TODO">our guide</Link> for more details.
+              </Typography>
+              <ErrorWrapper
```

Comment:
There may be an argument to move this `ErrorWrapper` and the one on line 68 into the `ListManager` component.

I decided not to do this as - 
 - It's only being used here
 - Scope creep?
 - The PR has more than enough going on as-is

Just flagging as this is something we can revisit if we start repeating this pattern or refactoring Editor components.